west.yml

```diff
@@ -26,7 +26,7 @@ manifest:
   # Please add items below based on alphabetical order
   projects:
     - name: cmsis
-      revision: 421dcf358fa420e9721a8452c647f0d42af8d68c
+      revision: 5d18367d9425a278c5931ed7dc0c4491e98150b1
```

Comment:
Can we set this to `revision: pull/10/head` to get testing of that PR.

Reply:
Thanks! I was searching this now :)